action/rlp_tx.go

```diff
@@ -34,15 +34,17 @@ func rlpSignedHash(tx rlpTransaction, chainID uint32, sig []byte) (hash.Hash256,
 	if len(sig) != 65 {
 		return hash.ZeroHash256, errors.Errorf("invalid signature length = %d, expecting 65", len(sig))
 	}
-	if sig[64] >= 27 {
-		sig[64] -= 27
+	sc := make([]byte, 65)
```

Comment:
not necessary, input is `sealed.Signature()`, which already did make([]byte), copy